communication/src/dtls_message_channel.cpp

```diff
@@ -17,17 +17,19 @@
  ******************************************************************************
  */
 
+#include "logging.h"
+LOG_SOURCE_CATEGORY("message_channel.dtls")
```

Comment:
I was hoping to not have to hard code the parent category, but I do like the idea of knowing the logs are a sub category of the communication library. Is it fair to say all files within the communication/ folder will have a default parent category of "comm" ?  That's probably a better way of keeping log messages filterable by "comm" as well ;-)  I'll make this change in a couple places.